pom.xml

```diff
@@ -16,9 +16,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.fluo</groupId>
-    <artifactId>fluo-io-parent</artifactId>
-    <version>2</version>
+    <groupId>org.apache.fluo</groupId>
+    <artifactId>fluo-parent</artifactId>
+    <version>1-SNAPSHOT</version>
```

Comment:
After parent POM release, this will be 1-incubating. Right now, it's 1-SNAPSHOT.